lib/config.ts

```diff
@@ -26,6 +26,7 @@ type SystemConfig = {
   DEFENDER_API_SECRET_MAINNET: string;
   ALCHEMY_ARBITRUM_KEY: string;
   ALCHEMY_ARBITRUM_TESTNET_KEY: string;
+  WALLETCONNECT_PROJECT_ID: string;
```

Comment:
Looks like we still need to add a value for this in vercel for production. Just flagging so we don't forget.